### PR TITLE
Create whats_next.md

### LINKED ISSUE #2: "What's next" document to be migrated to github and markdown
The following URL https://slack-files.com/T07EFKXHR-F4A137J0J-64e23a461f contains our "What's next" document, this needs to be translated to Markdown format.

### PR DESCRIPTION
**Pull request by joesanford**
Fixes #2

### CODE REVIEW

File: whats_next.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+https://www.hackerrank.com/  
+https://projecteuler.net/  
+Some of these may also incidentally help you to find a job.
+
+Make sure to iterate over your code and don't be afraid to ask if you are stuck. We will try to not tell you the answer outright, but we can point out flaws in your design.
+
+## Join the community :family:
+It might look scary at first, but where a project will help you grasp new packages and ideas, and where challenges will help you learnt o perfect your problem solving, the community is what will teach you the best practices and standards for your code.
+
+The easiest way is always to ask questions and request code reviews in #code-review. We will try to help as much as we can and offer advice on what you are doing and how to improve. If you can you should also feel free to contribute and answer questions yourself to help ground and confirm your own knowledge as well.
```
Comment (mattrasband) [inline whats_next.md]: Mind deep-linking to our `#help` channel? code-review is gone

File: whats_next.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Open Source Projects
+This is part of joining the community, but also bridges the gap between the community and starting your own project (and to some extent, taking a challenge).
+
+Joining an open source project can be one of the fastest ways to learn the best practices while simultaneously learning on a new project. Any mistakes you make that need to be corrected will be done when your code is reviewed and as a side effect you will learn about Version Control Systems, Pull Requests, Code Reviews, Unit testing and many other standard practices in the software world.
+
+Even more importantly, you will make a significant and lasting impact on the programming community which will help the community to develop as a whole. It will also look awesome on your CV/résumé.
+
+If you want to find a project to join, take a look at #community-projects where we run some of our own projects. You can even start your own and get some of us to join if we're interested.
```
Comment (mattrasband) [inline whats_next.md]: Deep link please with the pep8 naming!

**Review by mattrasband — CHANGES_REQUESTED** [review]

**Comment by joesanford** [comment]
Yeah should have dropped those in when ported to MD. All updated.

**Comment by ovv** [comment]
I wonder if it's not best to rename this file to something like `intermediate.md` so we could create a `beginner.md` ? Or something in this idea ?

**Comment by mattrasband** [comment]
Probably a good idea, maybe a python journey folder and we can put beginner/intermediate/advanced in there

**Comment by joesanford** [comment]
Maybe since the document isn't too long yet, we should just split it into different sections (beginner, intermediate, etc.), with links at the top of the document to them in a table of contents?

**Comment by mattrasband** [comment]
Yeah, that's probably good for now until we have more content :)

**Comment by ovv** [comment]
We should also make a link with the learning ressource repo. 

But we are going a bit out if scope for this PR

**Review by mattrasband — APPROVED** [review]

**Pull request merged**